tests/cmdline/commands/test_plugin.py

```diff
@@ -0,0 +1,43 @@
+"""Tests for the `verdi plugin list` command."""
+import pytest
+
+from aiida.cmdline.commands import cmd_plugin
+from aiida.plugins import CalculationFactory, WorkflowFactory
+
+
+def test_plugin_list(run_cli_command):
+    """Test the `verdi plugin list` command."""
+    from aiida.plugins.entry_point import ENTRY_POINT_GROUP_TO_MODULE_PATH_MAP
+
+    # Call base command without parameters and check that all entry point groups are listed
+    result = run_cli_command(cmd_plugin.plugin_list, [])
+    for key in ENTRY_POINT_GROUP_TO_MODULE_PATH_MAP:
+        assert key in result.output
+
+
+def test_plugin_list_group(run_cli_command):
+    """Test the `verdi plugin list` command for entry point group."""
+    from aiida.plugins.entry_point import ENTRY_POINT_GROUP_TO_MODULE_PATH_MAP
+
+    # Call for each entry point group and just check it doesn't except
+    for key in ENTRY_POINT_GROUP_TO_MODULE_PATH_MAP:
+        run_cli_command(cmd_plugin.plugin_list, [key])
+
+
+@pytest.mark.parametrize(
+    'entry_point_string', (
+        'aiida.calculations:arithmetic.add',
+        'aiida.workflows:arithmetic.multiply_add',
+        'aiida.workflows:arithmetic.add_multiply',
+    )
+)
+def test_plugin_list_detail(run_cli_command, entry_point_string):
+    """Test the `verdi plugin list` command for specific entry points."""
+    from aiida.plugins.entry_point import parse_entry_point_string
+
+    entry_point_group, entry_point_name = parse_entry_point_string(entry_point_string)
+    factory = CalculationFactory if entry_point_group == 'aiida.calculations' else WorkflowFactory
+    entry_point = factory(entry_point_name)
+
+    result = run_cli_command(cmd_plugin.plugin_list, [entry_point_group, entry_point_name])
+    assert entry_point.__doc__ in result.output
```

Comment:
Shouldn't these two be exactly equal?

Reply:
no, because the output of the command adds a lot of other things such as a formatted version of the spec